ocaml/xapi/xapi_pool.ml

```diff
@@ -2037,6 +2037,11 @@ let disable_ssl_legacy = set_ssl_legacy_on_each_host ~value:false
 
 let enable_ssl_legacy = set_ssl_legacy_on_each_host ~value:true
 
+let set_igmp_snooping_enabled ~__context ~self ~value = 
+  info "setting IGMP snooping %b" value;
```

Comment:
I'd remove the log line here, because we've already got one in message_forwarding immediately before we get here.